metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/GroupedInfluxDbHttpSender.java

```diff
@@ -0,0 +1,29 @@
+package com.izettle.metrics.influxdb;
+
+import java.util.concurrent.TimeUnit;
+
+import com.izettle.metrics.influxdb.data.InfluxDbWriteObject;
+import com.izettle.metrics.influxdb.utils.InfluxDbWriteObjectSerializer;
+
+/**
+ *  Passthrough to ultimately select a different style of serializer: grouped fields on one influxdb protocol line,
+ *  instead of one field per protocol line.
+ */
+public class GroupedInfluxDbHttpSender extends InfluxDbHttpSender {
```

Comment:
Could you write a bit extensive docs here, especially document the assumptions and limitations. (For example how tags are working).

After that this is ready for merge.

Reply:
Yes.  Was on break, will do!

Reply:
@mnp any luck getting this updated? This would really help me out, thanks!